adapter/src/main/kotlin/org/javacs/ktda/classpath/ProjectClassesResolver.kt

```diff
@@ -12,6 +12,8 @@ internal class ProjectClassesResolver(private val projectRoot: Path) : ClassPath
 		sequenceOf("kotlin", "java").flatMap { language ->
 			sequenceOf("main", "test").map { sourceSet ->
 				resolveIfExists(projectRoot, "build", "classes", language, sourceSet)
+				// kotlin multiplatform project jvm build path
+				resolveIfExists(projectRoot, "build", "classes", language, "jvm", sourceSet)
 			}
```

Comment:
This won't work since now the first expression will be ignored. Try something like this:

```suggestion
			sequenceOf("main", "test").flatMap { sourceSet ->
				sequenceOf(
					resolveIfExists(projectRoot, "build", "classes", language, sourceSet),
					// kotlin multiplatform project jvm build path
					resolveIfExists(projectRoot, "build", "classes", language, "jvm", sourceSet)
				)
			}
```

Note that this isn't a long-term solution either and we should probably find something better than hardcoding those paths.

Reply:
Sorry, thanks. The sequenceOf.flatMap+ sequenceOf looks correct. Was just kind of firing blind here without a way to test.

Looking at the "gradle properties" command line invocation, one can find the build directory. However, for multiplatform projects, that is still `buildDir: /Volumes/Dev/scratch/core/build`. There's no runtime `jvm` hint.